### ISSUE TITLE
[Feature Request] LoL Account overview

### ISSUE THREAD

**Issue by CiriousJoker** [TcNobo/TcNo-Acc-Switcher#37]
In a perfect world, here would be my flow:

1. Open "LoL Accounts" from Desktop
2. See a list of my accounts together with their SoloQ & Flex Queue rating and a list of champions (entered manually)
3. Click on the account to launch the game

I could build such a feature as a website. Clicking on an account opens a "tnco://MySummonerName" link (like Steam with their steamapps:// protocol). Building this as a website has the benefit of being decoupled and the protocol handler would allow easy integration with this tool.

**Comment by TcNobo** [comment]
Sounds good. The Battle.Net switcher has an Overwatch SR below the username, as seen in the image below. It's automatically collected from playoverwatch.com. There could be one built into the program in a similar way, although Riot seems very private with API/stats, and you need to request access...

I have previously wanted to set up a protocol for it, such as tcno://, but never got around to it. If it's simple enough I'll push it in an update today, as I needed to fix the Battle.Net switcher not showing SRs
![image](https://user-images.githubusercontent.com/10319195/123075640-b40d3480-d418-11eb-8db3-f75e1268fee0.png)

**Comment by TcNobo** [comment]
This feature was added with today's updates and is now in the latest release of the program https://github.com/TcNobo/TcNo-Acc-Switcher/commit/9c7a5f78faf52339593eb3a5ab9cf863d6e4288a.
It works similarly to the existing CLI. Create a shortcut to an account to see how it works. I will write up a Wiki page when I have some time to document the CLI and new protocol support. The reason I mention the CLI is that it's almost exactly the same.

**To switch a Steam account, use:**
`tcno:\\s:76561198064588130`
A breakdown: `tcno:\\` is the protocol, `s` is the platform, `:` breaks the platform and the account, and finally, the account's identifier which for Steam is the SteamID - `76561198064588130`

**For switching Epic Games accounts**, for example, use:
`tcno:\\e:AccountNameHere`
This time the platform is `e` for `E`pic Games, and the account's identifier is user-defined.

Creating a shortcut to the account, and looking at the account's name in the properties window of the shortcut itself will be the easiest way to copy/paste these into your own protocol usage. I may update shortcuts to use this, instead of pointing to the program and using the CLI -- in fact I will. This is a good idea.

**This protocol should be added by the installer.** Users who don't install will need to go into their settings in the TcNo Account Switcher and check `Enable tcno:\\ protocol (Recommended)` for this feature to be added to their registry, this is mainly for portable copies rather than static installations.

**Comment by TcNobo** [comment]
With all of that about adding protocol support aside, the League rating and more:
It's a good idea. It's something we can add to the account switcher itself. If you want to build a third-party website, I'm all for it, but users will need to either upload a file from the `LoginCache` folder or enter their account usernames again - as well as TcNo Account Switcher usernames.

As for building into the account switcher, iR3turnZ did it with the BattleNet account switcher and Overwatch. All you need is an API of sorts to check for stats. The PlayOverwatch API used is a website where info is read from and relies on the user providing their correct "BattleTag" as the name in the TcNo Account Switcher. The same can be done for League, Valorant and others.
I did mention Riot being strict with API access, as I already thought about doing something with Valorant ratings, but an API key is needed. There don't seem to be public sources where info like this is held, such as with PlayOverwatch.

tldr: If you know how already: Awesome. Let's work together on adding this, or at least making it easier for you to use with a third-party website.

**Comment by TcNobo** [comment]
Closing this issue for now, but do let me know if the project goes anywhere. I'm interested in hearing from you.

Since the last message another user reached out to me saying they created a plugin integrating the TcNo Account Switcher with the Wox launcher. https://github.com/aburakayaz/Wox.Plugin.SteamAccountSwitcher
- Though this was before I added the [Protocol](https://github.com/TcNobo/TcNo-Acc-Switcher/wiki/CLI-&-Protocol) support. I'd prefer the project to use this as it doesn't require a hardcoded install location, as long as it's installed it works - other than reading the files from the Documents (now `%AppData%\TcNo Account Switcher` folder). Using `tcno:\\....` would be much better. Anyways, unrelated.

I've featured it on the Discord under #community-projects, and will soon on the [Wiki](https://github.com/TcNobo/TcNo-Acc-Switcher/wiki) as well.

**Comment by robnjess** [comment]
![IMG_0199](https://user-images.githubusercontent.com/119259974/204196719-be06276c-5c44-4037-a9dc-3352917a6d97.jpg)
